File: src/crowstream/user/account/typeDefs.js
export const accountTypeDef = `
    type Account {
        id: String!
        email: String!
        is_email_verified: Boolean
    }
    type Token {
        token: String!
    }
    input AccountCredentials {
        email: String!
        password: String!
    }
`;

export const accountQueries = ``;

export const accountMutations = `
    signin(accountCredentials: AccountCredentials!): Token!
    signup(accountCredentials: AccountCredentials!): Account!
`;

File: api_gateway/schema/user/account/type_defs.py
import graphene

class Account(graphene.ObjectType):
    id = graphene.NonNull(graphene.String, name='id')
    email = graphene.NonNull(graphene.String, name='email')
    is_email_verified = graphene.Boolean(name='is_email_verified')


class Token(graphene.ObjectType):
    token = graphene.NonNull(graphene.String, name='token')
    refreshToken = graphene.String(name='refreshToken')


class AccountCredentials(graphene.InputObjectType):
    email = graphene.NonNull(graphene.String, name='email')
    password = graphene.NonNull(graphene.String, name='password')